tensorflow_lite_support/ios/task/audio/core/audio_tensor/sources/TFLAudioTensor.h

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "tensorflow_lite_support/ios/task/audio/core/audio_record/sources/TFLAudioRecord.h"
+#import "tensorflow_lite_support/ios/task/audio/core/sources/TFLRingBuffer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** A wrapper class to store input audio used in on-device machine learning. */
+NS_SWIFT_NAME(AudioTensor)
+@interface TFLAudioTensor : NSObject
+
+/** Audio format specifying the number of channels and sample rate supported. */
+@property(nonatomic, readonly) TFLAudioFormat *audioFormat;
+
+@property(nonatomic, readonly) TFLRingBuffer *ringBuffer;
```

Comment:
Please add docstring

Reply:
Besides, I think we shouldn't expose the TFLRingBuffer directly but only expose the TFLFloatBuffer of that ring buffer. It is to align with the Java API.

Reply:
I have made the ring buffer a private instance variable and added a `TFLFloatBuffer` and bufferSize property to get the elements